### PR TITLE
[skip ci]set cita_build container as bridge model

### PR DESCRIPTION
**Pull request by rink1969**

### CODE REVIEW

**Comment by rink1969** [comment]
@leeyr338 帮忙测试一下mac上的情况

File: env.sh [hunk 1]
```diff
@@ -4,21 +4,25 @@
...
 if test -f "${SOURCE_DIR}/Cargo.toml"; then
+    readonly SYSTEM_NET='bridge'
     readonly CONTAINER_NAME="cita_build${CONTAINER_NAME_HASH}"
     readonly DOCKER_IMAGE='cita/cita-build:ubuntu-18.04-20191128'
 else
+    if [[ "$(uname)" == 'Darwin' ]]; then
+        readonly SYSTEM_NET='bridge'
+        EXPOSE='1337:1337'
+    else
+        readonly SYSTEM_NET="host"
```
Comment (ouwenkg) [inline env.sh]: No need to specify `EXPOSE='1337:1337'` in unix system?

Comment (rink1969) [reply env.sh]: Yes, in unix system network is host.

Comment (leeyr338) [reply env.sh]: okay, it works!

**Pull request merged**